internal/pkg/skuba/addons/kured_test.go

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2019 SUSE LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package addons
+
+import (
+	"regexp"
+	"testing"
+
+	"github.com/SUSE/skuba/pkg/skuba"
+	"k8s.io/apimachinery/pkg/util/version"
+)
+
+func TestGetKuredImage(t *testing.T) {
+	tests := []struct {
+		name     string
+		imageTag string
+		want     string
+	}{
+		{
+			name:     "TestGetKuredImage",
+			imageTag: "1.2.0-rev4",
+			want:     skuba.ImageRepository + "/kured:1.2.0-rev4",
+		},
+	}
+	for _, tt := range tests {
+		tt := tt // Parallel testing
+		t.Run(tt.name, func(t *testing.T) {
+			if got := GetKuredImage(tt.imageTag); got != tt.want {
+				t.Errorf("GetKuredImage() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func Test_renderContext_KuredImage(t *testing.T) {
+	tests := []struct {
+		name          string
+		renderContext renderContext
+		want          string
+	}{
+		{
+			name: "Test_renderContext_KuredImage_1.15.2",
+			renderContext: renderContext{
+				config: AddonConfiguration{
+					ClusterVersion: version.MustParseSemantic("1.15.2"),
```

Comment:
Suggest to leverage `supportedVersions` instead of hard code versions.
For your reference: https://github.com/SUSE/skuba/blob/master/internal/pkg/skuba/kubernetes/versions_test.go#L97

Reply:
changed